model/registry/db.yaml

```diff
@@ -529,3 +533,67 @@ groups:
           reference the [elasticsearch schema](https://raw.githubusercontent.com/elastic/elasticsearch-specification/main/output/schema/schema.json)
           in order to map the path part values to their names.
         examples: ['db.elasticsearch.path_parts.index=test-index', 'db.elasticsearch.path_parts.doc_id=123']
+  - id: registry.db.vector
+    prefix: db.vector
+    type: attribute_group
+    brief: >
+      This group defines attributes for vector databases.
+    attributes:
+      - id: similarity
```

Comment:
suggesting more explicit name like `db.vector.search.similarity_metric`

Reply:
@ezimuel I think you also missed this change, is marked as resolved without any updates :)

Reply:
Fixed in https://github.com/open-telemetry/semantic-conventions/pull/1231/commits/a3330ffea04ad5d2ce8d9a4bb07a1ce259369339

Reply:
BTW, who/how/when is going to populate it? 

It seems it's only available at index creation time and not available at query or insert time. I.e. very rarely.

Reply:
Correct, this is generally provided during the index creation but some databases uses it also in the query (e.g. [Qdrant](https://qdrant.tech/documentation/concepts/search/#metrics)).

I think we can also leverage the [Conditionally Required](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) level for some attributes that are not always available, like the `db.vector.search.similarity_metric`, WDYT?

Reply:
@lmolkova just a reminder for this, thanks.

Reply:
Thanks, it makes sense! 

BTW I think the attribute should be `db.search.similarity_metric` or similar - similarity is not always based on vectors and we don't need to limit this attribute.